css/pages/layout/css-grid.css

```diff
@@ -0,0 +1,20 @@
+.grid {
+  background: var(--COLOR-LIME-200);
+  border-radius: 0.2rem;
+  padding: 1rem;
+}
+
+.grid > div {
+  background: var(--COLOR-LIME-500);
+  border-radius: 0.2rem;
```

Comment:
avoid converting values less than 10px to rem's as suggested by Vikram

Reply:
Thanks for pointing this out, Shubham. Actually, for consistency, I am using rem everywhere and Vikram suggested that it is not necessary to convert values less than 10px into rem, as they will not affect a lot on different media. But, I think, he didn't mention that it will harm a lot if we will use rems for a value less than 10px.

Please correct me if I am wrong.